test/test_sparse_csr.py

```diff
@@ -1338,6 +1338,20 @@ def run_test_block_addmm_addmv(self, addmv_addmm, c, a, b, op_b=False, op_out=Fa
         self.assertEqual(actual, out)
         self.assertEqual(actual, expected)
 
+        if addmv_addmm == torch.addmm:
+            # test dense x bsc by transposing the entire test case
+            a_t = a.transpose(-2, -1)
+            b_t = b.transpose(-2, -1)
+            c_t = c.transpose(-2, -1)
+            actual_t = addmv_addmm(c_t, b_t, a_t, alpha=alpha, beta=beta)
+            out_t = torch.empty_like(out.transpose(-2, -1))
+            addmv_addmm(c_t, b_t, a_t, alpha=alpha, beta=beta, out=out_t)
+
+            expected_t = alpha * (b.cpu().resolve_conj().numpy().transpose() *
+                                  a_bsr.transpose()) + beta * c.cpu().numpy().transpose()
```

Comment:
Should this be:
```suggestion
            expected_t = alpha * (b.cpu().resolve_conj().numpy().transpose() @
                                  a_bsr.transpose()) + beta * c.cpu().numpy().transpose()
```
?

Reply:
I was confused by this too, it appears that for `scipy.sparse.sp_matrix` subclasses (which `b_bsr` is `__mul__` is matrix multiplication. 

I will see about changing it.